'URLSearchParams' is not defined

**What version of ESLint are you using?** 

2.4

**What parser (default, Babel-ESLint, etc.) are you using?** 

babel-eslint ^6.0.0-beta.6

**Please show your full configuration:** 

eslint-config-airbnb 6.1.0

**What did you do? Please include the actual source code causing the issue.**

`const searchParams = new URLSearchParams();`

**What did you expect to happen?**

`URLSearchParams ` should not be flagged as undefined because it is part of WebAPIs

**What actually happened? Please include the actual, raw output from ESLint.**


 13:30  error  'URLSearchParams' is not defined  no-undef

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
     "esutils": "^2.0.2",
     "file-entry-cache": "^1.1.1",
     "glob": "^7.0.3",
-    "globals": "^8.18.0",
+    "globals": "^9.2.0",
     "ignore": "^3.0.10",
     "imurmurhash": "^0.1.4",
     "inquirer": "^0.12.0",

--- a/tests/lib/rules/no-undef.js
+++ b/tests/lib/rules/no-undef.js
@@ -53,6 +53,7 @@ ruleTester.run("no-undef", rule, {
         { code: "var a; ({a}) = {};", parserOptions: { ecmaVersion: 6 } },
         { code: "var a; ({b: a}) = {};", parserOptions: { ecmaVersion: 6 } },
         { code: "var obj; [obj.a, obj.b] = [0, 1];", parserOptions: { ecmaVersion: 6 } },
+        { code: "URLSearchParams;", env: { browser: true } },
 
         // Notifications of readonly are removed: https://github.com/eslint/eslint/issues/4504
         { code: "/*global b:false*/ function f() { b = 1; }" },

